src/configs/websocket/index.ts

```diff
@@ -15,12 +15,17 @@ export type TDerivApi = {
   authorize: (requestData: AuthorizeRequest) => Promise<AuthorizeResponse>;
 };
 
+let attempts = 10;
+const RECONNECT_INTERVAL = attempts * 10000;
```

Comment:
Are these numbers part of the task's requirements?

Reply:
It was not mentioned precisely, I saw in deriv-app we are reconnecting with a constant interval of 5000ms.